src/index.js

```diff
@@ -29,7 +32,10 @@ async function start(fields) {
   const bills = await getBillsList()
   await saveBills(bills, fields, {
     identifiers: ['bipandgo'],
-    contentType: 'application/pdf'
+    contentType: 'application/pdf',
+    sourceAccount: this.accountId,
+    sourceAccountIdentifier: fields.login,
```

Comment:
nit: you can use this.saveBills to not have to specify sourceAccount and sourceACcountIdentifier : https://github.com/konnectors/libs/blob/5999b72798381e644e342f126dc6f52bae9144a8/packages/cozy-konnector-libs/src/libs/BaseKonnector.js#L445-L451